Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md

```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+---
+
+# Customize Your theme
+
+A custom theme lets you add a little personal touch to your page - the colors you want, the font, and all that. If you are new to working with Umbraco Uno and do not yet know how to work with the theme then it is recommended that you [take a look.](../../Getting-Started/Themes/index.md)
+
+## How to customize your theme through the theme editor
+
+We will be working with the theme editor, , which you can access by going to the preview section of the _Unicorn Hotel_ page.
+
+### Picking the colors and text of the theme
+
+First what we will need to do, is click the green button that looks like a paintbrush - then the theme editor will show up.
+The theme we will be setting up will be a watermelon theme, and it will look like this:
+
+![watermelon theme](images/theme.gif)
+
+:::note
+If you want a colored theme you will have to go to the Hero Widget and set the overlay to 0 otherwise the color you choose for background color will be put on top of your hero image
+::::
+
+1. Select ***Theme*** and press in the field and type in the value you see on this image:
+
+    ![watermelon theme color](images/Color-settings.png)
+
+2. Do this for all the fields and press ***Save***
+3. This will result in the color Scheme for the dark theme to be set up like intended.
+4. Select ***Text*** in the menu bar at the top of the theme editor.
+5. Choose Roboto as both Heading and Body Font
```

Comment:
Removed item number 2, as I've suggested to write the first bullet.
Removed item number 3, as it wasn't really a step, but more a statement.

```suggestion
2. Select ***Text*** in the menu bar at the top of the theme editor.
3. Choose Roboto as both Heading and Body Font
```